release-content/0.14/release-notes/12013_Upstreaming_bevy_color.md

```diff
@@ -1 +1,31 @@
-TODO
+Colors are a huge part of building a good game: UI, effects, shaders and more all need fully-feature, correct and convenient color tools.
+Bevy now supports a wide range of color spaces, each with their own type (e.g. `LinearRgba`, `Hsla` or `Oklaba`),
```

Comment:
```suggestion
Bevy now supports a broad selection of color spaces, each with their own type (e.g. `LinearRgba`, `Hsla` or `Oklaba`),
```

Nit: Mild rewording because "_wide range_" is used again at a latter point in the sentence.